### ISSUE TITLE
User Defined Object Issue

### ISSUE THREAD

**Issue by saung87** [robgridley/pace-api#51]
Hello,

   After I created a new user-defined object and try to insert value to the attribute of that UDO, it says like this: "**Type must be CapitalizedWords**". What is that error means and how to solve it? 

Thanks.

**Comment by bhartvigsen** [comment]
We've run into the same problem. We commented out this line in Model.php in our fork:

```
if (!preg_match('/^([A-Z]+[a-z]*)+$/', $type)) {
        throw new InvalidArgumentException('Type must be CapitalizedWords');
}
```

I'm not clear what the original intent of this line is. I think it's meant to validate objects but UDOs are not accounted for.

We've been running without it with no ill effects as far as I can tell.

**Comment by saung87** [comment]
Thank you for your reply @bhartvigsen . it is ok for now. for my case, i made some changes to the regular expression as follows and it works.
`if (!preg_match('/^([\W\S]+[A-Z]+[a-z]*)+$/', $type)) {
            throw new InvalidArgumentException('Type must be CapitalizedWords');
        }`

**Comment by robgridley** [comment]
I've removed the check from the Model constructor and released it as v2.0.3.
